lib/http_client.js

```diff
@@ -89,6 +100,13 @@ class HttpClient extends EventEmitter {
         }
     }
 
+    _setSessionCookie(cookie) {
+        if (cookie.startsWith('session=')) {
```

Comment:
I m wondering if we should also check that the value of cookie is a string (and not undefined in case it was empty array as from the code above).

Reply:
That is possible although if the set-cookie header is returned all it can contain is string, or arrays of string. If the array was empty the reference to the first item would have failed

Reply:
Actually, reference to a non-existent array item would return undefined.